engines/director/castmember/bitmap.cpp

```diff
@@ -585,6 +587,19 @@ void BitmapCastMember::load() {
 					}
 
 					debugC(5, kDebugImages, "BitmapCastMember::load(): Bitmap: id: %d, w: %d, h: %d, flags1: %x, flags2: %x bytes: %x, bpp: %d clut: %s", imgId, surf->w, surf->h, _flags1, _flags2, _bytes, _bitsPerPixel, _clut.asString().c_str());
+
+					if (ConfMan.getBool("dump_scripts")) {
+
+						Common::String prepend = "stream";
+						Common::String filename = Common::String::format("./dumps/%s-%s-%d.png", encodePathForDump(prepend).c_str(), tag2str(tag), imgId);
+						Common::DumpFile screenshotFile;
```

Comment:
This name is misleading. It is not a `screenshotFile` but rather a `bitmapFile`; please rename it accordingly. Same for the variable name below